### PR TITLE
Periodic Table Docs

### PR DESCRIPTION
**Pull request by dgasmith**
Adds periodic table docs and tries out another way of documenting APIs.

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -20,27 +22,47 @@ Pip
...
+or use ``pip`` for a local install:
+
+.. code-block:: console
+
+    >>> pip install -e .
+
+
+Testing
+-------
+QCElemental can be tested once install using the ``pytest`` package which can be installed via Conda as well:
```
Comment (loriab) [inline docs/source/install.rst]: drop "once install"

File: docs/source/periodic_table.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    >>> qcel.periodictable.to_mass(6)
+    12.0
+    >>> qcel.periodictable.to_mass("C")
+    12.0
+    >>> qcel.periodictable.to_mass("C12")
+    12.0
+    >>> qcel.periodictable.to_mass("Carbon")
+    12.0
+
+A brevity of infomation can be accessed in this manner. Taking 'Carbon' as an example for mass information:
```
Comment (loriab) [inline docs/source/periodic_table.rst]: brevity --> variety?

Comment (dgasmith) [reply docs/source/periodic_table.rst]: I was thinking bevy, but variety is better.

Comment (loriab) [reply docs/source/periodic_table.rst]: either :-)

File: docs/source/periodic_table.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
+Periodic Table
+==============
+
+Full access to the periodic table is also available within QCPortal. This
```
Comment (loriab) [inline docs/source/periodic_table.rst]: I don't care but is QCPortal what you meant?

Comment (dgasmith) [reply docs/source/periodic_table.rst]: Nope. Make a patch?

**Pull request merged**